packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/create/iti39-envelope.ts

```diff
@@ -21,6 +22,12 @@ export type BulkSignedDR = {
   outboundRequest: OutboundDocumentRetrievalReq;
 };
 
+function wrapDocUniqueIdIfLowercaseUuid(docUniqueId: string): string {
+  const valid = validate(docUniqueId);
+  const lowercase = docUniqueId === docUniqueId.toLowerCase();
+  return valid && lowercase ? wrapIdInUrnUuid(docUniqueId) : docUniqueId;
+}
```

Comment:
I'm concerned we're building logic for a specific GW and applying it to all of them, so I just want to make sure this works for all.

- `validate` can we rename the import to `validateUuid` and call the const `isValidUuid`?
- can we rename `lowerCase` to `isLowerCase`?
- instead of using valid and lowercase to define whether to wrap it around UUID URN, can we check if there's no UUID URN and then wrap it?